src/penn_chime/models.py

```diff
@@ -66,14 +65,13 @@ def __init__(self, p: Parameters):
             intrinsic_growth_rate = get_growth_rate(p.doubling_time)
 
             self.beta = get_beta(intrinsic_growth_rate,  gamma, self.susceptible, 0.0)
+            self.beta_t = get_beta(intrinsic_growth_rate, self.gamma, self.susceptible, p.relative_contact_rate)
 
             self.i_day = 0 # seed to the full length
-            self.beta_t = self.beta
-            self.run_projection(p)
+            self.run_projection(p, [(self.beta, p.n_days)])
```

Comment:
This ignores mitigation entirely. That's not a change from before this PR, but it is a deficiency.

Reply:
Opened #439